experimental/packages/opentelemetry-instrumentation/src/platform/browser/index.ts

```diff
@@ -14,4 +14,4 @@
  * limitations under the License.
  */
 
-export * from './instrumentation';
+export { InstrumentationBase } from './instrumentation';
```

Comment:
These changes are not really needed for this change, but as I was fixing this test I updated to removed the `export * from` syntax.